data/templates/bootstrap.js

```diff
@@ -273,4 +273,22 @@ function getFile(downloadUrl, cb) {
     }).end();
 }
 
+/**
+ * Check valid accepted response code - check whether the code is an array of number
+ * @private
+ * @param code
+ */
+function checkValidAcceptCode(code) {
+    if (!(code instanceof Array)) {
+        return false;
+    }
+
+    return code.every(function(item) {
+        if (typeof item !== 'number') {
```

Comment:
Even further, if the `acceptResponseCodes` is a single valid number, not an array, we can internally wraps this number to an array (` 1 --> [1]`), this should be more user friendly. I proposal this becomes I remembered for OS bootstrap, users often input a single vland ID and then cause rendering error, because actually we required it as an Array of number. 

Of course, this kinds of conversion can be better done in on-tasks who publish the command.

I'm OK to do this or not.